test/suite/test_backup08.py

```diff
@@ -125,6 +125,7 @@ def backup_and_recover(self, expected_rec_ts):
         self.pr("query recovery ts: " + q)
         self.assertTimestampsEqual(q, timestamp_str(expected_rec_ts))
 
+    @unittest.skip("Temporarily disabled")
```

Comment:
These tests aren't going to be possible in the future, are they? Don't they have to eventually go away?

Reply:
The majority of them should be possible in the future, with rollback to stable implemented and perhaps with some small adjustments to the test themselves. So i'm hesitant to delete them entirely just yet.